test/ttmlir/Silicon/TTMetal/simple_constant.mlir

```diff
@@ -0,0 +1,15 @@
+// RUN: ttmlir-opt --ttir-load-system-desc="path=%system_desc_path%" --ttir-to-ttmetal-backend-pipeline %s > %t.mlir
+// RUN: FileCheck %s --input-file=%t.mlir
+// RUN: ttmlir-translate --ttmetal-to-flatbuffer %t.mlir > %t.ttm
+// UNSUPPORTED: true
```

Comment:
@ddilbazTT I think you can now use this PR and remove this line to generate a simple test for runtime support.